modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -110,18 +111,18 @@ To avoid a conflict when running multiple proxies using the same domain, ensure
 
 === API Gateway Domain Template
 
-If you choose to download your API proxy and configure it manually on your end, you may use the *API Gateway domain template*.
+If you choose to download and manually configure your API proxy on your end, use the *API Gateway domain template*.
```

Comment:
delete "on your end" (unless the reader also has an api proxy to manually configure on some other end).

Reply:
Done! Thanks!